src/Model/Join.php

```diff
@@ -136,6 +137,13 @@ class Join
     public function __construct(string $foreign_table = null)
     {
         $this->foreign_table = $foreign_table;
+
+        // handle foreign table containing a dot - that will be reverse join
+        if (strpos($this->foreign_table, '.') !== false) {
+            // split by LAST dot in foreign_table name
+            [$this->foreign_table, $this->foreign_field] = preg_split('~\.+(?=[^.]+$)~', $this->foreign_table);
+            $this->reverse = true;
```

Comment:
Can we still overwrite $reverse property while initializing join?

Reply:
yes